### PR TITLE
Update csharp examples code to latest top-level statements

### PR DESCRIPTION
**Pull request by Zaid-Ajaj**
This PR updates (most of) the csharp example code to latest convention from .NET 6: using top-level statement programs which simplify program definition (no need for MyStack.cs nor explicit output type decorations)

Continuation of [pulumi-hugo#1656](https://github.com/pulumi/pulumi-hugo/pull/1656)

### CODE REVIEW

**Review by mikhailshilkov — APPROVED** [review]
LGTM. Naive question - were all these examples written manually or generated with programgen at some point?

File: themes/default/content/registry/packages/akamai/_index.md [hunk 1]
```diff
@@ -112,24 +112,20 @@ using Pulumi;
...
-            });
-        });
-}
+    var group = GetGroup.Invoke();
+    var contract = GetContract.Invoke();
+
+    var hostname = new Properties.EdgeHostName("demo", new Properties.EdgeHostNameArgs
+    {
+        Contract = contract.Id,
+        Group = group.Id,
```
[the remark this answers is not in the document]
Comment (stack72) [reply themes/default/content/registry/packages/akamai/_index.md:123]: These will have been manually written :/

[the remark this answers is not in the document]
Comment (stack72) [reply themes/default/content/registry/packages/akamai/_index.md:123]: We can merge this and then I will open an issue to recreate these with convert and yaml

[the remark this answers is not in the document]
Comment (Zaid-Ajaj) [reply themes/default/content/registry/packages/akamai/_index.md:123]: @mikhailshilkov Yeah you are right, I only rewrote the program structure, not the code underneath which indeed looks buggy. When we update program-gen for .NET and use it with YAML, it should generate much better code here!

**Comment by Zaid-Ajaj** [comment]
Will merge this as-is for the time being to reflect .NET 6 changes. The current code for each package is currently hand-written but should be generated from PCL or YAML in the future

**Pull request merged**